example-get-started/generate.sh

```diff
@@ -135,6 +143,20 @@ dvc stage add -n train \
   -o model.pkl \
   python src/train.py data/features model.pkl
 dvc repro
+python <<EOF
+from dvc.repo import Repo
+from dvc.annotations import Artifact
+
+repo = Repo(".")
+artifact = Artifact(
+  path="model.pkl", 
+  type="model",
+  desc="Detect whether the given stackoverflow question should have R language tag",
+  labels=["nlp", "classification", "stackoverflow"]
+)
+repo.artifacts.add("text-classification", artifact)
+EOF
+
```

Comment:
Why not change the source code to use `live.log_artifact()`?

Reply:
Right 😂 Not sure how I missed that. Will do!